### ISSUE TITLE
Request for documentation about the difference between this and https://github.com/chocolatey/chocolatey-oneget

### ISSUE THREAD

**Issue by CMCDragonkai** [jianyunt/ChocolateyGet#1]
Basically in the title.

Is this provider better?

After testing it for a bit, I found that it worked in 2 cases where the original prototype chocolatey provider didn't work. One that involved 7zip archiving, and the other that had a `-` in its package name.

Can we know where binaries and command line executables are stored?

**Comment by CMCDragonkai** [comment]
@jianyunt I noticed that after using Chocolatey then ChocolateyGet upon installation of a package via ChocolateyGet, all of the former packages in `C:\Chocolatey` has been moved to `C:\ProgramData\chocolatey`. Even `Get-Packages` says those packages are now part of `ChocolateyGet`. However the existing `ChocolateyPath` environment variable did not get updated to `C:\ProgramData\chocolatey`. However there's a new system environment variable called `ChocolateyInstall` which does point to `C:\ProgramData\chocolatey`.

According to http://stackoverflow.com/a/28239451 this is a change that occurred in Chocolatey itself. Therefore I think `ChocolateyPath` is an environment variable that gets deprecated.

**Comment by CMCDragonkai** [comment]
This also means that using ChocolateyGet is actually more compatible with using Choco.exe directly, it gives you access to it. In fact just by using it, gives you access to it, there's no need to install chocolatey as a package directly. And also it's possible to install Chocolatey provider packages afterwards.

**Comment by jianyunt** [comment]
You are right. ChocolateyGet is a wrapper of Choco.exe, so installation stuff is all handled by choco.exe.

**Comment by jianyunt** [comment]
Regarding the difference https://github.com/chocolatey/chocolatey-oneget  and ChocolateyGet:

ChocolateyGet is a [OneGet](https://oneget.org) provider. I myself created it sometime back when I see there are some needs from users ( I do not own Chocolatey, but I own the OneGet). As I mentioned, it is just a wrapper around the choco.exe.  Operations like search, install, or installationpath etc are all handled by choco.exe.  

https://github.com/chocolatey/chocolatey-oneget  is owned by chocolatey owner (Rob?).  I noticed there are significant work to be done to this  provider and Rob was tied up with something else.  For the time-being, I wrote ChocolateyGet .
